lms/djangoapps/instructor/management/commands/collect_ora2_data.py

```diff
@@ -33,7 +38,7 @@ def handle(self, *args, **options):
         except InvalidKeyError:
             raise CommandError("The course ID given was invalid")
 
-        file_name = ("%s-ora2.csv" % course_id).replace("/","-")
+        file_name = ("%s-ora2.csv" % course_id).replace("/", "-")
```

Comment:
I know you didn't write this, but since we're touching it...

We prefer to use the string `format` method, instead of `%` based interpolation.

```python
file_name = "{course_id}-ora2.csv".format(
    course_id=course_id,
)
file_name = file_name.replace('/', '-')
```

Additionally, by splitting it out into multiple assignment statements, it's easier to step through this code with a debugger and monitor state change.